### DISCUSSION TITLE
Clear rules for releasing next versions

### DISCUSSION THREAD

**Discussion by slawekjaranowski** [support-and-care/maven-support-and-care#25] [Ideas for Funding]
The one of most important for collaborators is to know when they can expect release next version of component which they did a contribution. 

When user did a PR with fix, new feature and next would like to use it in their projects ... so waiting a long time can be a little frustrated and not encourages for next contributions.

**Comment by khmarbaise** [comment]
Good point, but that would require a kind of release schedule which might be not OK for some of the Maven Committers/Contributors etc. ? Maybe some kind of roadmap ? Or a time based release schedule (like the JDK does) of course with shorter time period?

**Comment by rmannibucau** [comment]
Any point discussing it, got rejected every time it entered the dev list so feel like it will be loosing time to discuss it again.
Maybe just associate it to a trigger ("if there is a CVE") for ex?
